### PR TITLE
Formbricks Branding Signature

### PR DESCRIPTION
**Pull request by jobenjada**
## What does this PR do?

Formbricks Branding Signature added (optional)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

In Preview 
grafik

grafik

Link Survey
grafik

Widget
grafik

Settings (will get an update at some point)
grafik

## Checklist

Please test it through once :)

- [X] I ran `pnpm build` and `pnpm lint` to prevent build and linting errors
- [X] My contribitiom does not cause any warnings
- [X] I have removed all `console.logs`
- [X] I have merged the latest changes from main onto my branch with `git pull origin main`

### CODE REVIEW

File: packages/js/src/components/FormbricksSignature.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { h } from "preact";
+
+export default function FormbricksSignature() {
+  return (
+    <a
+      href="https://formbricks.com?utm_source=survey_branding"
+      target="_blank"
+      className="fb-mb-5 fb-mt-2 fb-flex fb-justify-center">
+      <p className=" text-xs text-slate-400">
```
Comment (mattinannt) [inline packages/js/src/components/FormbricksSignature.tsx:9]: please prefix classes with `fb-` and remove space

**Review by mattinannt — CHANGES_REQUESTED** [review]
@jobenjada Currently the migration sets `formbricksSignature` to true (for all existing users) and the schema was altered to a default value of `false` but no migration was created.
Please fix to achieve the desired behaviour, e.g. first migration default false, afterwards a migration that changes default value to true (for new users).

**Review by mattinannt — CHANGES_REQUESTED** [review]
@jobenjada Currently the migration sets `formbricksSignature` to true (for all existing users) and the schema was altered to a default value of `false` but no migration was created.
Please fix to achieve the desired behaviour, e.g. first migration default false, afterwards a migration that changes default value to true (for new users).

**Comment by jobenjada** [comment]
hoi :) added migrations and resolved merge conflicts 🙌

**Comment by jobenjada** [comment]
Resolved merge conflicts. PR contains 2 migrations, ready from my side 🚀

**Review by mattinannt — APPROVED** [review]
good job 😊💪🎉

**Pull request merged**